rad/src/types/array.rs

```diff
@@ -118,77 +121,69 @@ impl fmt::Display for RadonArray {
 
 impl Operable for RadonArray {
     fn operate(&self, call: &RadonCall) -> Result<RadonTypes, RadError> {
+        self.operate_in_context(call, &mut ReportContext::default())
+    }
+
+    fn operate_in_context(
+        &self,
+        call: &RadonCall,
+        context: &mut ReportContext<RadonTypes>,
+    ) -> Result<RadonTypes, RadError> {
+        let wip0022 = context
+            .active_wips
+            .as_ref()
+            .map(ActiveWips::wip0022)
+            .unwrap_or_else(|| true);
```

Comment:
```suggestion
            .unwrap_or(true);
```

Clippy should have detected this

Reply:
Fixed :wink: